spring-integration-redis/src/test/java/org/springframework/integration/redis/store/RedisMessageGroupStoreTests.java

```diff
@@ -400,16 +403,21 @@ void testJsonSerialization() {
 		Message<?> mutableMessage = new MutableMessage<>(UUID.randomUUID());
 		Message<?> adviceMessage = new AdviceMessage<>("foo", genericMessage);
 		ErrorMessage errorMessage = new ErrorMessage(new RuntimeException("test exception"), mutableMessage);
+		Message<?> delayMessage = new DefaultMessageBuilderFactory()
+				.withPayload(new DelayHandler.DelayedMessageWrapper(genericMessage, System.currentTimeMillis()))
+				.copyHeaders(genericMessage.getHeaders())
+				.build();
```

Comment:
If we go `@JsonCreator` way, then this could be changed to the:
```
		var delayedMessageWrapperConstructor =
				BeanUtils.getResolvableConstructor(DelayHandler.DelayedMessageWrapper.class);
		Message<?> delayMessage = new GenericMessage<>(
				BeanUtils.instantiateClass(delayedMessageWrapperConstructor, genericMessage, System.currentTimeMillis()));
```